website/docs/d/action_rule_profile.html.markdown

```diff
@@ -39,3 +39,6 @@ data "aci_action_rule_profile" "example" {
 * `annotation` - (Optional) Annotation of object Action Rule Profile.
 * `name_alias` - (Optional) Name Alias of object Action Rule Profile.
 * `description` - (Optional) Description of object Action Rule Profile.
+* `set_communities` - (Optional) Set Communities of object Action Rule Profile.
```

Comment:
Document what are the expected map attributes and add type Map to the documentation.